Update README.md

GvandeSteeg opened the pull request:
Do we want to simplify it this far or keep it as is? I don't mind either way, but then I know how to approach the others

GvandeSteeg committed:
Update README.md

trstickland commented:
Users _should_ use `docker_deploy.sh` as an argument for that ssh command _unless_ we have restricted their ssh access (as privately documented in fce-management;  I didn't document it here as it's an internal matter concerning how we manage access to our infrastructure).

Bear in mind that if a user with restricted ssh access does use the `docker_deploy.sh` argument (as documented here prior to this PR), then what will be executed on the box is:
```
docker_deploy.sh  docker_deploy.sh  <version>  <docker user>
```
[sic]
i.e. the script will see "docker_deploy.sh" as the value of the first argument it is passed.   I figured this would happen sooner or later [so I wrote the script to tolerate that](https://github.com/sanger-pathogens/bactgen-training-app/blob/master/docker_deploy.sh#L11-L14).

I recommend leaving the README unchanged (i.e. drop this PR), as those instructions will work for people whether or not their ssh access is restricted.

trstickland commented:
I did though notice something else that should really be changed.

The command specifies `<docker user>` as "the name of the docker repository you plan to use".   I think using `<docker registry>` as placeholder text would be better, as it's not really  a _user_.

(I would also prefer _docker registry_ to _docker repository_, personally.)

GvandeSteeg committed:
updates

undo removing of `docker_deploy.sh`
renaming <docker user> to <docker registry>

GvandeSteeg commented:
Agreed and updated
